src/classes/Countertop.js

```diff
@@ -43,8 +44,48 @@ class Countertop {
 		return station
 	}
 
-	updateTopology = () => {
+	/**
+	 * Start the countertop and begin data processing.
+	 *
+	 * This will regenerate the countertop topology to reflect all currently added appliances.
+	 *
+	 * @return {Boolean} Whether the countertop successfully started.
+	 */
+	start = async () => {
+		this.logger.debug('CountertopCoordinator: start()')
+		await this.updateTopology()
```

Comment:
We should move updateTopology to the `addAppliance` -- that way the topology is always assumed to reflect the appliances added.